packages/esm-patient-appointments-app/src/routes.json

```diff
@@ -33,10 +33,6 @@
         "path": "Appointments"
       }
     },
-    {
-      "name": "appointments-form-workspace",
-      "component": "appointmentsFormWorkspace"
-    },
```

Comment:
this now gets set up in the esm-appointments-app

Reply:
Hi @mogoodrich!
Can you please register workspace with `registerWorkspace` instead of `routes.json` in the `esm-appointments-app`?
Thanks!
![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/51502471/afbe98b1-62ff-41b1-8e6e-a468d3c95d38)

Reply:
A bit tangential. I was under the impression that the workspace stuff only works in `esm-patient-chart` given the nowhere in `esm-patient-management` do we use `registerWorkspace` and `launchPatientWorkspace` as of now. I would be really happy if that's not the case.

(The above probably still works for patient chart regardless)

Reply:
The availability of this functionality is the result having the patient-common-lib as a peerDependency of the module (which is what implements those functions). The only advantage the patient chart has is that it will implicitly depend on the latest version.